inventory/s3/orc_reader.go

```diff
@@ -72,13 +72,13 @@ func (r *OrcInventoryFileReader) inventoryObjectFromRow(rowData []interface{}) I
 		isDeleteMarker = swag.Bool(rowData[isDeleteMarkerIdx].(bool))
 	}
 	return InventoryObject{
-		Bucket:         rowData[r.orcSelect.IndexInSelect["bucket"]].(string),
-		Key:            rowData[r.orcSelect.IndexInSelect["key"]].(string),
-		Size:           size,
-		LastModified:   lastModified,
-		Checksum:       eTag,
-		IsLatest:       isLatest,
-		IsDeleteMarker: isDeleteMarker,
+		Bucket:             rowData[r.orcSelect.IndexInSelect["bucket"]].(string),
+		Key:                rowData[r.orcSelect.IndexInSelect["key"]].(string),
+		Size:               size,
+		LastModifiedMillis: lastModified,
```

Comment:
Why do we need `millis` here?  Alternatively, if `lastModified` is already `int64` milliseconds, can we either convert it to `time.Time` when created, or at least call it `lastModifiedMillis`?